models.py

```diff
@@ -106,6 +106,9 @@ class Round(db.Model):
     started_at = db.Column(db.DateTime, nullable=False)
     ended_at = db.Column(db.DateTime, nullable=True)
 
+    sample_size_options = db.Column(db.String(1000), nullable=True)
```

Comment:
I was wondering why we didn't use the JSON data type. SQLAlchemy supports it. 
There's an awful lot of `json.dumps` happening around the code to keep this a string.